### DISCUSSION TITLE
Can I Use Copilot Individual License for Business / Organization

### DISCUSSION THREAD

**Discussion by ss-aditya** [community/community#51476] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi All,

I just start using Copilot this few days and start using the trial version.  I want to propose the Copilot for my company to use, and can we use Individual License for business purposes? 

Thanks n Regards,
SS

**Comment by Akash1134** [comment]
Hi @ss-aditya ,

If you have paid for an individual copilot subscription it up to you or your employer to negotiate if they will permit you to use it while in their employment, we cannot provide any guidance.

However, if your employ would like to take out a Copilot subscription they can invite you to the organisation which will cancel your existing individual subscription and refund you the balance.

Your Code is subject to the GitHub [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies). For example, you may not prompt GitHub Copilot with content that is unlawful or otherwise prohibited by the GitHub Acceptable Use Policies on GitHub.com.

**Reply by vkoivula** [reply]
@Akash1134 could you link and quote the exact place where commercial use or personal licses is prohibited? I was asked about this, but I could only find this your answer and can't seem to find it being mentioned in any links you provide. Thanks.

**Reply by Masterxilo** [reply]
This answer does not address the question. It merely states that from the perspective of github, copilot enterprise version is used to enable the use of github copilot to everyone in a github "organization" or "team"... the OP question was about applicability of copilot to commercial projects. I doubt that Microsoft buried a clause somewhere that prevents usage of any version of copilot for commercial purposes. They just have a few save-our-asses statements saying you retain full responsibility for your code including accepted suggestions and that they use telemetry on your code unless you disable it and that copyrighted/licensed code might be suggested unless you explicitly filter/disable it here: https://github.com/settings/copilot

edit: the answer was changed after I wrote this it's fine now and reflects these observations

**Reply by cervenf** [reply]
Hi,
Does that mean there is no legal restriction to use "Copilot Individual License" inside a company? We don't have to go with Copilot Business ?
Thank you.

**Comment by ss-aditya** [comment]
Hi There,

Thanks for your information about the licensing.

Thanks n Regards,

**Comment by Akash1134** [comment, marked as answer]
Hi everyone, 

If you have paid for an individual copilot subscription it is up to you or your employer to negotiate if they will permit you to use it while in their employment, we cannot provide any guidance.

However, if your employ would like to take out a Copilot subscription they can invite you to the organisation which will cancel your existing individual subscription and refund you the balance.

Thanks!

**Reply by gkees** [reply]
Hi,
Is this 100% correct that GitHub will automatically cancel an individual subscription to Copilot (if they already have it) when being invited to an organization's Copilot plan and signing up that way?

**Reply by BenjamenMeyer** [reply]
This answer fails to provide insight into liability for an org that chooses to accept an individual's license should there be an issue - information an organization should know in order to be able to make a proper determination.

**Comment by ss-aditya** [comment]
Hi Akash,

Thank you for your information about the Subscription.

Best Regards,
Santo
